src/NHibernate.Test/Async/NHSpecificTest/GH0750/Fixture.cs

```diff
@@ -0,0 +1,115 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by AsyncGenerator.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+
+using System.Collections.Generic;
+using NHibernate.Criterion;
+using NHibernate.Transform;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH0750
+{
+	using System.Threading.Tasks;
+	using System.Threading;
+
+	[TestFixture]
+	public class FixtureAsync : BugTestCase
```

Comment:
Its generation was forgotten. See [this section of contributing](https://github.com/nhibernate/nhibernate-core/blob/master/CONTRIBUTING.md#regenerate-async-code).

Reply:
OOPS! It was very hectic to work with VS2017 since it keeps throwing itself into NOT RESPONDING state for my install while I add/edit/rename files in test project. Would keep it in mind if needed to write another one.

Reply:
No troubles. Yes VS2017 is currently quite bad at handling large projects migrated to its new csproj format. That is why I had mentioned Rider, which does quite better. But maybe things will start to be better with changes like Microsoft/msbuild#2572.

About writing another one, you will need to clean your master branch. But you should do it only after this PR is closed, otherwise you will need to open a new one from a dedicated branch this time. 

For cleaning, in case you need instructions, it could be something like:

 * Branch away changes done on master if you want to keep them. Assuming you are currently on master, `git branch newBranch`.
 * Still on master, reset it hard `git reset --hard head~x`, where x is the number of commits to remove (2 currently `git reset --hard head~2`).
 * Force push it to your remote `git push -f`.

Reply:
Thanks for the detailed instructions. I will wait for this PR to close.